PlayTools/MysticRunes/PlayedApple.swift

```diff
@@ -0,0 +1,247 @@
+//
+//  PlayWeRuinedIt.swift
+//  PlayTools
+//
+//  Created by Venti on 16/01/2023.
+//
+
+import Foundation
+import Security
+
+// Implementation for PlayKeychain
+// World's Most Advanced Keychain Replacement Solution:tm:
+// This is a joke, don't take it seriously
+
+public class PlayKeychain: NSObject {
+    static let shared = PlayKeychain()
+
+    private static func getKeychainDirectory() -> URL? {
+//        // Get the keychain folder
+//        let keychainFolder = FileManager.default.urls(for: .documentDirectory,
+//                                                          in: .userDomainMask)
+//            .first?.appendingPathComponent("Keychain")
+//
+        let bundleID = Bundle.main.infoDictionary?["CFBundleIdentifier"] as? String ?? ""
+        let keychainFolder = URL(fileURLWithPath: "/Users/\(NSUserName())/Library/Containers/io.playcover.PlayCover")
+            .appendingPathComponent("PlayChain")
+            .appendingPathComponent(bundleID)
+
+        // Create the keychain folder if it doesn't exist
+        if !FileManager.default.fileExists(atPath: keychainFolder.path) {
+            do {
+                try FileManager.default.createDirectory(at: keychainFolder,
+                                                        withIntermediateDirectories: true,
+                                                        attributes: nil)
+            } catch {
+                if PlaySettings.shared.settingsData.playChainDebugging {
+                    NSLog("BUZZINC: Failed to create keychain folder")
+                    }
```

Comment:
Trailing curly brackets for debugging is indented wrongly all around the code
Btw how about extracting these debugging code to new method?
```Swift
@objc static public func playChainDebug(_ message: String) {
     if PlaySettings.shared.settingsData.playChainDebugging {
         NSLog(message)
     }
 }
```

Reply:
Done, for both new files